src/libraries/System.Diagnostics.Process/tests/ProcessTests.Unix.cs

```diff
@@ -1016,3 +1049,13 @@ private static string StartAndReadToEnd(string filename, string[] arguments)
         }
     }
 }
+
+// TODO: Use the actual localization data in here?
```

Comment:
Please help me to figure out what to do with this.

Apparently, the `Interop.*.cs` files I've added to the test assembly relies on resources/localization, but I haven't figured out how to reuse the main projects' resources in tests.

I think it's possible to just duplicate the API I use in tests here and get rid of `SR`. But this will be quite a lot of duplication, since the structure returned by `proc_pid_rusage` is kinda long.

Reply:
That is a great question!

You should be able to reference the resource file in the test project similar to:

https://github.com/dotnet/runtime/blob/913231acc359c5e94d4397e8fe0c8a38ae6b0c5a/src/libraries/System.Formats.Tar/tests/System.Formats.Tar.Tests.csproj#L5

Reply:
For some reason, this only started working for me after I added OSX-specific TFM to the test project. I guess it has something to do with how the resource layout works? Anyway, please take another look at the current solution. I have to say I am a bit confusing around this part.